scripts/generate-version-info.sh

```diff
@@ -20,4 +20,9 @@ echo $(jq ".binaries.kubelet = \"$(kubelet --version | awk '{print $2}')\"" $OUT
 echo $(jq ".binaries.awscli = \"$(aws --version | awk '{print $1}' | cut -d '/' -f 2)\"" $OUTPUT_FILE) > $OUTPUT_FILE
 
 # cached images
-echo $(jq ".images = [ $(sudo ctr -n k8s.io image ls -q | cut -d'/' -f2- | sort | uniq | grep -v 'sha256' | xargs -r printf "\"%s\"," | sed 's/,$//') ]" $OUTPUT_FILE) > $OUTPUT_FILE
+if systemctl is-active --quiet containerd; then
+  echo $(jq ".images = [ $(sudo ctr -n k8s.io image ls -q | cut -d'/' -f2- | sort | uniq | grep -v 'sha256' | xargs -r printf "\"%s\"," | sed 's/,$//') ]" $OUTPUT_FILE) > $OUTPUT_FILE
+elif [ "${CACHE_CONTAINER_IMAGES}" = "true" ]; then
+  echo "containerd must be active to generate version info for cached images"
```

Comment:
How will this behave in older k8s versions with docker cri?

Reply:
There's no behavior difference. If `cache_container_images` is enabled, then we'll include the image details here. We don't enable this feature on versions that use Docker, though.